openwisp_monitoring/device/api/views.py

```diff
@@ -248,5 +265,31 @@ def _create_clients_graph(self, metric):
         graph.full_clean()
         graph.save()
 
+    def _create_resources_graph(self, metric, resource):
+        graph = Graph(metric=metric, configuration=resource)
+        graph.full_clean()
+        graph.save()
+
+    def resources_metric_write(self, pk, ct, key, value):
+        if key not in monitoring_settings.AUTO_GRAPHS:
+            return
+        metric, created = Metric._get_or_create(
+            object_id=pk,
+            content_type=ct,
+            key=key,
+            field_name=key,
```

Comment:
Using field_name=key=key is not very useful when storing the disk usage data. Here is my proposal, let me know what you think:

* name: "mountpoint /" (using key "mounted_on")
* field_name: "used_space" and  "size" (save these 2 metrics)
* key "/dev/sda1" (using key "name")

Reply:
Good idea for `disk_usage`, I can use the `field_name` say `used_memory` (and similarly `shared` and `buffered` when adding them) for `memory`. Any suggestions, what can I name the `field_name` for the `metric` saving `load` data.

Reply:
Yeah, load_1, load_5 and load_15 look good to me